src/gui/touchscreengui.cpp

```diff
@@ -909,7 +919,8 @@ void TouchScreenGUI::translateEvent(const SEvent &event)
 
 		if (m_move_id != -1) {
 			if ((event.TouchInput.ID == m_move_id) &&
```

Comment:
This line hasn't been touched, but nevertheless

```suggestion
			if (event.TouchInput.ID == m_move_id &&
```

Reply:
Done.